src/status_im/contexts/wallet/create_account/view.cljs

```diff
@@ -40,9 +40,18 @@
     :image             :icon
     :image-props       :i/derivated-path
     :action            :button
-    :action-props      {:on-press    #(js/alert "Coming soon!")
+    :action-props      {:on-press    (fn []
+                                       (rf/dispatch [:standard-auth/authorize
+                                                     {:on-auth-success
+                                                      (fn [password]
+                                                        (rf/dispatch [:navigate-to
+                                                                      :screen/wallet.edit-derivation-path
+                                                                      {:customization-color
+                                                                       customization-color
+                                                                       :password password}]))
+                                                      :auth-button-label (i18n/label :t/continue)}]))
```

Comment:
Maybe it's a good moment to extract the inner dispatch to a `let` (and also maybe the outer one) and give it a name.